src/OrbitGl/ThreadTrack.cpp

```diff
@@ -61,7 +61,7 @@ ThreadTrack::ThreadTrack(CaptureViewElement* parent, TimeGraph* time_graph,
 
 std::string ThreadTrack::GetThreadNameFromTid(uint32_t thread_id) {
   const std::string kEmptyString;
```

Comment:
We can also get rid of this variable now.

Reply:
Yes, thanks.